aiida/tools/importexport/common/progress_bar.py

```diff
@@ -15,7 +15,7 @@
 
 from aiida.common.lang import type_check
 
-from aiida.tools.importexport.common.config import BAR_FORMAT
+from aiida.common.progress_reporter import TQDM_BAR_FORMAT as BAR_FORMAT
```

Comment:
The progress_bar.py file will be removed when the import refactoring PR is merged, correct?

Reply:
yep thats the intention 👍